### ISSUE TITLE
ICLR 2020 | WHY  GRADIENT  CLIPPING  ACCELERATES  TRAINING:ATHEORETICAL JUSTIFICATION FOR ADAPTIVITY

### ISSUE THREAD

**Issue by richardbaihe** [richardbaihe/paperreading#64]
This paper investigate why adaptive gradient methods outperform SGD for some NLP tasks, e.g. language modeling. 
They find the gradient smoothness and gradient norm could be linear correlated with large learning rate and clipped gradient.

**Comment by richardbaihe** [comment]
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/11585856/110277257-e2b6ea80-7fa2-11eb-8502-ed8c2143f856.png">
